### PR TITLE
updated faq.jsx, index,jsx

### PR DESCRIPTION
**Pull request by RickDeb2004**
Just check the import section, I have not done correctly that
review my code I will change as per the review @Shurtu-gal

### CODE REVIEW

File: src/components/molecules/faq.jsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+`;
+
+const Answer = styled.div`
+  font-size: 22px;
+  padding-top: 16px;
+  display: ${(props) => (props.active ? 'block' : 'none')};
+`;
+
+const FaqComponent = () => {
+  const faqs = [
```
Comment (Shurtu-gal) [inline src/components/molecules/faq.jsx]: Put data in the data folder for easier updation afterwards.

File: src/components/molecules/faq.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  };
+
+  return (
+    <FaqContainer>
+      <FaqHeading>Faqs</FaqHeading>
+      <Caption>Ask whatever you want</Caption>
+      <FaqLayout>
+        {faqs.map((faq, index) => (
+          <FaqItem
+            className={activeIndex === index ? 'active' : ''}
```
Comment (Shurtu-gal) [inline src/components/molecules/faq.jsx]: What if I want to open up multiple faqs?

File: src/components/molecules/faq.jsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const FaqContainer = styled.div`
```
Comment (Shurtu-gal) [inline src/components/molecules/faq.jsx]: Put styles in separate styles.jsx

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

File: src/components/molecules/faq.jsx (hunk 2 above)
Comment (RickDeb2004) [reply src/components/molecules/faq.jsx]: ya I have fixed this one I have not commited that yet...I have checked that , that's working

Comment (RickDeb2004) [reply src/components/molecules/faq.jsx]: commited

File: src/components/molecules/faq.jsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+};
+
+  return (
+    <FaqContainer>
+      <FaqHeading>Faqs</FaqHeading>
+      <Caption>Ask whatever you want</Caption>
+      <FaqLayout>
+        {faqs.map((faq, index) => (
+          <FaqItem
+            className={activeIndex.includes(index) ? 'active' : ''}
```
Comment (Shurtu-gal) [inline src/components/molecules/faq.jsx]: Put this in another component with their own state. It would make it better.

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

File: src/components/molecules/faq.jsx (hunk 4 above)
Comment (RickDeb2004) [reply src/components/molecules/faq.jsx]: @Shurtu-gal  bhaiya I just updated above those changes and now for this I have to make  suppose "faqItem" component , so should I add this faqItem.js file in the atom folder?

Comment (RickDeb2004) [reply src/components/molecules/faq.jsx]: @Shurtu-gal bhaiya just for  now I just updated faq.js in data folder I will import that in final review , and also added the style.js file in molecule , at the time of final review I will remove the style components from faq.jsx file right?.....

Comment (Shurtu-gal) [reply src/components/molecules/faq.jsx]: > @Shurtu-gal bhaiya I just updated above those changes and now for this I have to make suppose "faqItem" component , so should I add this faqItem.js file in the atom folder?

Yeah do that 🚀

Comment (RickDeb2004) [reply src/components/molecules/faq.jsx]: @Shurtu-gal  done ... added FaqItem.jsx in atoms....
Now come to final review....
1.I have to import the faq.js from data folder in faq.jsx
2.I have to comment out the style code in faq.jsx as I have already included style.js file .
3.I have to comment out the style code in FaqItem.jsx file 

Right?

File: src/components/molecules/faq.jsx (hunk 3 above)
Comment (RickDeb2004) [reply src/components/molecules/faq.jsx]: done

File: src/components/molecules/faq.jsx (hunk 4 above)
Comment (RickDeb2004) [reply src/components/molecules/faq.jsx]: as per mine above comment point 2 and point 3 done

File: src/components/atoms/faq/FaqItem.jsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+//   > div:first-child { 
+//     text-align: left; 
+//   }
+// `;
+
+// const Answer = styled.div`
+//   font-size: 22px;
+//   padding-top: 16px;
+//   display: ${(props) => (props.active ? 'block' : 'none')};
+// `;
```
Comment (ashutosh-rath02) [inline src/components/atoms/faq/FaqItem.jsx]: Remove these comments

File: src/components/molecules/faq/faq.jsx [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+// text-align: left; 
+// padding-right: 25px; 
+// margin-bottom:160px;
+
+// @media (max-width: 768px) {
+//   font-size: 36px;
+//   padding-right: 15px; 
+//   margin-bottom:80px;
+// }
+// `;
```
Comment (ashutosh-rath02) [inline src/components/molecules/faq/faq.jsx]: These too!!

File: src/components/molecules/faq/faq.jsx [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+//   }
+// `;
+
+// const Answer = styled.div`
+//   font-size: 22px;
+//   padding-top: 16px;
+//   display: ${(props) => (props.active ? 'block' : 'none')};
+// `;
+
+const FaqComponent = () => {
```
Comment (ashutosh-rath02) [inline src/components/molecules/faq/faq.jsx:5]: Why is this still here?

**Review by ashutosh-rath02 — CHANGES_REQUESTED** [review]

File: src/components/atoms/faq/FaqItem.jsx (hunk 5 above)
Comment (RickDeb2004) [reply src/components/atoms/faq/FaqItem.jsx]: yeah  I will

File: src/components/molecules/faq/faq.jsx (hunk 7 above)
Comment (RickDeb2004) [reply src/components/molecules/faq/faq.jsx:5]: yes

Comment (RickDeb2004) [reply src/components/molecules/faq/faq.jsx:5]: @Lucif3r-in @Shurtu-gal done, now check

File: src/pages/Playground/index.jsx [hunk 8]
```diff
@@ -2,6 +2,7 @@ import MentorCarousel from '../../components/molecules/MentorCarousel/MentorCaro
 import { Body2, Body3, Button, ButtonMeta, Heading1, Heading2, Heading3, Heading4, NavText } from '../../components/shared';
 import Body1 from '../../components/shared/Typography/Body1';
 import { Container } from './styles';
+import Faq from '../../components/molecules/faq/faq';
```
Comment (ashutosh-rath02) [inline src/pages/Playground/index.jsx:1]: Place the FAQ component in the playground for us to check.

File: src/components/atoms/faq/FaqItem.jsx [hunk 9]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FaqItemContainer, Answer} from './style';
+
+
+
+const FaqItem = ({ faq, isActive, onClick }) => {
+  return (
+    <FaqItemContainer className={isActive ? 'active' : ''} onClick={onClick}>
```
Comment (Shurtu-gal) [inline src/components/atoms/faq/FaqItem.jsx]: isActive state should be managed in each component. It would be easier for you also.

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

File: src/pages/Playground/index.jsx (hunk 8 above)
Comment (RickDeb2004) [reply src/pages/Playground/index.jsx:1]: should I place the faq component like 
  const Playground =()=>{
return (after the mentorcarousel should I write faq component)

is this pseudo code fit for this ?
@Lucif3r-in

File: src/components/atoms/faq/FaqItem.jsx (hunk 9 above)
Comment (RickDeb2004) [reply src/components/atoms/faq/FaqItem.jsx]: @Shurtu-gal   done  , check my this recent commit,
I will also delete the comments once I get the confirmation.
pls review this

File: src/pages/Playground/index.jsx (hunk 8 above)
Comment (RickDeb2004) [reply src/pages/Playground/index.jsx:1]: done @Lucif3r-in

File: src/components/molecules/faq.jsx (hunk 3 above)
Comment (RickDeb2004) [reply src/components/molecules/faq.jsx]: @Lucif3r-in @Shurtu-gal now I have to add an "+" icon each question , should I do that ri8 now?

**Comment by Shurtu-gal** [comment]
@RickDeb2004 run `npm run format`

**Comment by RickDeb2004** [comment]
@Shurtu-gal It returns error
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path C:\kerberos/package.json
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, open 'C:\kerberos\package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\dell\AppData\Local\npm-cache\_logs\2023-08-26T06_43_43_097Z-debug-0.log

**Comment by RickDeb2004** [comment]
> 

I have done 
1. npm install prettier --save-dev
2.the script also format:check
though its not working

**Comment by ashutosh-rath02** [comment]
> > 
> 
> I have done
> 
> 1. npm install prettier --save-dev
>    2.the script also format:check
>    though its not working

Use npm run format

**Comment by RickDeb2004** [comment]
@Lucif3r-in now check

**Comment by ashutosh-rath02** [comment]
Follow the design properly.
- [ ] Borders to each card
- [ ] Background of the cards
- [ ] Add up and down arrows use iconify
- [x] Use transition time, it is opening instantly and keep some larger text so that we can see how it looks for more text
@RickDeb2004

**Comment by RickDeb2004** [comment]
> Follow the design properly.
> 
> * [ done]  Borders to each card
> * [ done]  Background of the cards
> * [done ]  Add up and down arrows use iconify
> * [done ]  Use transition time, it is opening instantly and keep some larger text so that we can see how it looks for more text
>   @RickDeb2004

@Shurtu-gal @Lucif3r-in

**Comment by RickDeb2004** [comment]
@Lucif3r-in check my recent commit

**Comment by RickDeb2004** [comment]
@Lucif3r-in @Shurtu-gal check now

File: package.json [hunk 10]
```diff
@@ -20,6 +20,12 @@
     ]
   },
   "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@iconify/icons-mdi": "^1.2.47",
+    "@iconify/icons-uil": "^1.2.3",
+    "@mui/icons-material": "^5.14.6",
+    "@mui/material": "^5.14.6",
```
Comment (ashutosh-rath02) [inline package.json]: Remove these

Comment (RickDeb2004) [reply package.json]: @Lucif3r-in done

Comment (RickDeb2004) [reply package.json]: > @Lucif3r-in after removing all the checks are failed

Comment (RickDeb2004) [reply package.json]: @Lucif3r-in Deleting those dependencies created problem in code , that's why I am not deleting these 
i have also done npm uninstall  "@mui/icons-material": "^5.14.6",
    npm "@mui/material": "^5.14.6"
    but throws error

**Comment by RickDeb2004** [comment]
@Shurtu-gal @Lucif3r-in I have also changed the bg-color of question card from grey to white while it is active .

File: src/components/atoms/faq/FaqItem.jsx [hunk 11]
```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+import { FaqItemContainer, Answer, ArrowIcon } from './style';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
```
Comment (Shurtu-gal) [inline src/components/atoms/faq/FaqItem.jsx]: Use icons from iconify here that's why it's giving an error when uninstalling these.

**Review by Shurtu-gal — APPROVED** [review]
LGTM 🚀

**Pull request merged**